std/internal/math/biguintcore.d

```diff
@@ -1624,14 +1648,15 @@ body
     if (data.length > 1)
         data[1] = 0;
 
-    for (int i= (s[0]=='-' || s[0]=='+')? 1 : 0; i<s.length; ++i)
```

Comment:
By the time the string gets here, the sign is already popped from the range, see https://github.com/JackStouffer/phobos/blob/bigint2/std/bigint.d#L82